Portal/vip-application/src/main/java/fr/insalyon/creatis/vip/application/server/dao/mysql/ClassData.java

```diff
@@ -94,24 +76,13 @@ public void add(AppClass appClass) throws DAOException {
      */
     @Override
     public void update(AppClass appClass) throws DAOException {
+        
+        removeFromClass(appClass.getName(), "engine");
+        addToClass(appClass.getName(), appClass.getEngines(), "engine");
```

Comment:
Then, update means either adding engines or removing engines. Why don't we have an add and a remove method that can deal with a list of engines? It sounds like an overkill to remove all the engines and add the new ones.